src/sdkV2.ts

```diff
@@ -490,4 +522,14 @@ export default class CegaEvmSDKV2 {
     const oracleEntry = await this.loadOracleEntry();
     return oracleEntry.getPrice(baseAsset, quoteAsset, timestamp, oracleDataSource);
   }
+
+  async pythUpdateAssetPrices(
+    timestamp: number,
+    assetAddresses: EvmAddress[],
+    updates: string,
+    fee: number,
+  ): Promise<ethers.providers.TransactionResponse> {
+    const pythAdapter = await this.loadPythAdapter();
+    return pythAdapter.updateAssetPrices(timestamp, assetAddresses, updates, { value: fee });
```

Comment:
```
function updateAssetPrices(
        uint40 timestamp,
        address[] calldata assets,
        bytes[] calldata updateDatas
```

if addresses takes in an array, then `updates` should also be taking in an array